packages/react/src/app-layout/AppLayoutSidebarNav.tsx

```diff
@@ -72,17 +79,33 @@ function AppLayoutSidebarNavListItem({
 	item,
 }: AppLayoutSidebarNavListItemProps) {
 	const Link = useLinkComponent();
+	const { closeMobileMenu } = useAppLayoutContext();
 	const { endElement, icon: Icon, label, ...restItemProps } = item;
 
+	function onClick(event: MouseEvent<HTMLAnchorElement | HTMLButtonElement>) {
+		// Close the mobile menu when a link or button has been pressed
+		closeMobileMenu();
```

Comment:
does this occur on link presses or just onClick?

Reply:
It will happen on both 👍